src/assets/styles/var.sass

```diff
@@ -245,7 +245,7 @@ $app-color-action-list-item-inline-divider: #d0d7de7a
   background-color: $app-color-btn-counter-bg
 .btn:hover, .btn.hover, [open]>.btn
   background-color: $app-color-btn-hover-bg
-  border-color: $app-color-btn-hover-border
+  border-color: $app-color-btn-hover-border !important
```

Comment:
Please don't use `!important`

I wasn't able to find exact place there this rule is used, but if you need help to solve this - please contact me.